plugins/woocommerce-admin/client/marketing/utils/getRecommendationSource.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Internal dependencies
+ */
+import { getAdminSetting } from '~/utils/admin-settings';
+
+/**
+ * Get the source of the marketing recommendations.
+ *
+ * When the marketplace suggestions feature is turned on, the source is 'woocommerce.com'. Otherwise, it is 'plugin-woocommerce'.
+ */
+export const getRecommendationSource = () => {
+	if ( getAdminSetting( 'allowMarketplaceSuggestions', false ) ) {
+		return 'woocommerce.com';
```

Comment:
Any reason for using `woocommerce.com` and not `woo.com`?

Reply:
This is because currently all the woocommerce.com API calls are called with the `https://woocommerce.com/wp-json/...` URL. (You can try search for "woocommerce.com/wp-json" and "woo.com/wp-json" in the code base.)